tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -115,6 +115,8 @@ spec:
         - "$(params.REKOR_HOST)"
         # NOTE: The syntax below is required to negate boolean parameters
         - "--strict=$(params.STRICT)"
+        # Include rule descriptions and titles in the output
+        - "--info"
```

Comment:
I realized we can do `--info=$(params.INFO)` similar to strict. Will revise the PR.